recipes/dfvue/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "dfvue" %}
+{% set version = "0.993" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dfvue-{{ version }}.tar.gz
+  sha256: 6b2f33316d0d102e49f03b1b03aac65a2e6fd01bd7aa2a6d4fa415a77b53ab19
+
+build:
+  number: 0
+  noarch: python
```

Should fix the Windows failure.

```suggestion
  noarch: python
  gui_scripts:
    - dfvue = dfvue.__main__:main
```

---

I had a copy-paste there. It should be ok now